numpy/core/src/multiarray/einsum_sumprod.c.src

```diff
@@ -80,7 +80,7 @@ static NPY_GCC_OPT_3 @temptype@ @name@_sum_of_arr(@type@ *data, npy_intp count)
     /* Use aligned instructions if possible */
     const int is_aligned = EINSUM_IS_ALIGNED(data);
     const int vstep = npyv_nlanes_@sfx@;
-    npyv_@sfx@ vaccum = npyv_zero_@sfx@();
+    npyv_@sfx@ vacuum = npyv_zero_@sfx@();
```

Comment:
Please restore all vaccum to vaccum, or possibly change to v_accum.

Reply:
Yes, perhaps `v_accum` would be better.